lib/src/base-styles/_nx-page-header.scss

```diff
@@ -8,82 +8,92 @@
 @import '../scss-shared/nx-text-helpers';
 @import '../scss-shared/nx-container-helpers';
 
+$main-header-background: $nx-sonatype-blue;
+$main-header-color: $nx-off-white;
+$main-header-height: 72px;
+$main-header-product-height: 52px;
+$main-header-product-title-row-height: 2fr;
+$main-header-product-version-row-height: 1fr;
+
 /*
   #Main Header
 */
 
 .nx-header {
   background-color: $nx-off-black;
   flex-shrink: 0; // needed for IE11
-  height: $nx-main-header-height;
+  height: $main-header-height;
   width: 100%;
 }
 
 .nx-header__inner {
   @include container-horizontal;
 
   display: flex;
+  height: $main-header-height;
   margin: 0 auto;
   width: 100%;
   max-width: $nx-page-width-max;
 }
 
-.nx-title {
-  border-right: 1px solid transparent;
-  border-left: 1px solid transparent;
+.nx-product {
+  align-self: center;
   box-sizing: border-box;
-  color: #fff;
+  color: $nx-off-white;
   display: -ms-grid;
   display: grid;
-  grid-column-gap: 2px;
-  grid-row-gap: 2px;
-  -ms-grid-rows: 22px 2px auto;
-  grid-template-rows: 22px auto;
-  -ms-grid-columns: 36px 2px auto;
-  grid-template-columns: 36px auto;
+  grid-column-gap: $nx-spacing-xxs;
+  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
+  grid-template-columns: $main-header-product-height auto;
   grid-template-areas:
     "logo title"
     "logo version";
-  height: 40px;
-  padding-right: 12px;
+  height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
-.nx-app-logo {
-  -ms-grid-row-align: center;
-  align-self: center;
+.nx-product__logo {
   -ms-grid-row: 1;
-  -ms-grid-row-span: 3;
+  -ms-grid-row-span: 2;
   -ms-grid-column: 1;
+  -ms-grid-row-align: center;
+  align-self: center;
+  box-sizing: border-box;
   grid-area: logo;
-  height: 36px;
-  width: 36px;
+  height: 100%;
+  width: 100%;
 }
 
-.nx-title__brand {
-  @include semi-bold();
-
-  -ms-grid-row-align: end;
-  align-self: end;
-  font-size: 18px;
+.nx-product__name {
   -ms-grid-row: 1;
   -ms-grid-column: 3;
+  @include semi-bold();
+
+  -ms-grid-row-align: center;
+
+  align-self: center;
+  font-size: 23px;
   grid-area: title;
   -ms-grid-column-align: left;
   justify-self: left;
-  line-height: 18px;
+  line-height: normal;
+  text-transform: lowercase;
   white-space: nowrap;
 }
 
-.nx-title__version {
-  align-self: start;
-  -ms-grid-row-align: start;
-  font-size: 12px;
-  -ms-grid-row: 3;
+.nx-product__version {
+  -ms-grid-row: 2;
   -ms-grid-column: 3;
+  @include bold();
+
+  -ms-grid-row-align: start;
+  align-self: start;
+  font-size: 11px;
   grid-area: version;
   -ms-grid-column-align: left;
   justify-self: left;
-  line-height: 12px;
+  line-height: 100%;
```

Comment:
FYI descenders stick out of this box too

Reply:
> FYI descenders stick out of this box too

I switched to "normal" here: https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/1c003acbd02b1d00ec23906843d06370bf80ab99

Reply:
> > FYI descenders stick out of this box too
> 
> I switched to "normal" here: [1c003ac](https://github.com/sonatype/sonatype-react-shared-components/commit/1c003acbd02b1d00ec23906843d06370bf80ab99)

After trying "normal", "100%", and various numerical values, I opted for "1.1" as the best compromise between position and descender coverage. https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/6f321eb7ffa7069bc96911f210ed649c94b50e49